src/openrct2-ui/windows/Park.cpp

```diff
@@ -1498,7 +1498,10 @@ static void window_park_objective_mouseup(rct_window* w, rct_widgetindex widgetI
  */
 static void window_park_objective_resize(rct_window* w)
 {
-    window_set_resize(w, 230, 226, 230, 226);
+    if (gCurrentTTFFontSet == nullptr)
```

Comment:
This fails in DISABLE_TTF builds. In these builds, it should (obviously) always assume the sprite font.